x/bank/index.go

```diff
@@ -23,9 +23,6 @@ type IndexedKeeper struct {
 func NewIndexedKeeper(keeper *Keeper, accKeeper auth.AccountKeeper) *IndexedKeeper {
 	index := IndexedKeeper{Keeper: *keeper, accKeeper: accKeeper, accsToUpdate: make([]sdk.AccAddress, 0)}
 	hook := func(ctx sdk.Context, from sdk.AccAddress, to sdk.AccAddress) {
-		if ctx.IsCheckTx() {
```

Comment:
Why you've deleted this section?

Reply:
Cuz, as we know, handlers out of scope for tx check. So, during tx check, there is no way to reach here.